examples/complete/README.md

```diff
@@ -0,0 +1,53 @@
+This example demonstrates a terraform-generated Tamr config for a full aws-scale out environment set up for static Spark clusters. The environment consists of:
+- static EMR deployment running both HBase and Spark
+- data bucket and logs bucket shared by both HBase and Spark
+- newly-generated EC2 key pair (used by both Tamr VM and EMR EC2 instances)
+- Elasticsearch domain
+- RDS Postgres instance
+- Tamr VM deployment
+
```

Comment:
Could add the networking resources that are created to this list, since this is the distinguishing difference between the complete and minimal examples.